src/pages/users/Users.tsx

```diff
@@ -26,12 +36,70 @@ enum UserRole {
   Facilitator = "Facilitator",
 }
 
+const defaultUserData: UserRequest = {
+  email: "",
+  is_admin: false,
+};
+
 const Users = () => {
   const classes = useStyles();
   const { users } = useContext(UsersContext);
+  const [displayInviteDialog, setIsDisplayInviteDialog] = useState(false);
+  const [snackbarMessage, setSnackbarMessage] = useState("");
+  const [newUser, setNewUser] = useState(defaultUserData);
+
+  const handleOpenInviteDialog = () => {
+    setIsDisplayInviteDialog(true);
+  };
+
+  const handleCloseInviteDialog = () => {
+    setNewUser(defaultUserData);
+    setIsDisplayInviteDialog(false);
+  };
+
+  const convertBooleanToRole = (isAdmin: boolean) =>
+    isAdmin ? UserRole.Admin : UserRole.Facilitator;
+  const convertRoleToBoolean = (role: UserRole) => role === UserRole.Admin;
+
+  const inviteUser = async () => {
+    try {
+      await UserAPI.postUser(newUser);
+      handleCloseInviteDialog();
+      setSnackbarMessage(
+        `Successfully added ${newUser.email} to the application.`
```

Comment:
i believe the designs had this instead!
`Invite was successfully sent to ${newUser.email}`